content/en/docs/concepts/deployment-models/index.md

```diff
@@ -0,0 +1,421 @@
+---
+title: Istio Deployment Models
```

Comment:
```suggestion
title: Deployment Models
```

Reply:
@geeknoid has always argued (and I agree) that using Istio in any titles is redundant.

Reply:
How about something like "Advanced Deployment Models"? I'm concerned that naming this topic just "Deployment Models" will attract all audiences and introduce them to complexities they don't need to know yet 🤷‍♂ just an opinion

Reply:
Possibly, although the simple models are here too. I'm hoping that the content here isn't too advanced for any end-user to grok.